mlir/test/Integration/Dialect/SparseTensor/CPU/lit.local.cfg

```diff
@@ -11,6 +11,12 @@ if config.mlir_run_arm_sve_tests:
     config.substitutions.append(
         ("%VLA_ARCH_ATTR_OPTIONS", '--march=aarch64 --mattr="+sve"')
     )
+    config.substitutions.append(
+        (
+            "%mlir_native_utils_lib_dir",
+            config.arm_emulator_utils_lib_dir or config.mlir_lib_dir,
+        )
+    )
```

Comment:
this is no longer used? (and below at the end of the `else`)

Reply:
nice catch. I also updated comments about UNSUPPORTing the tests on Windows.